jetty-core/jetty-server/src/test/java/org/eclipse/jetty/server/CustomRequestLogTest.java

```diff
@@ -1030,6 +1030,21 @@ public boolean handle(Request request, Response response, Callback callback)
         assertThat(log, is("value1234"));
     }
 
+    @Test
+    public void testLogBadUserAgent() throws Exception
+    {
+        start("User-Agent: %{User-Agent}i, \"%{User-Agent}i\"");
+
+        HttpTester.Response response = getResponse("""
+            GET / HTTP/1.0
+            User-Agent: bad"value
```

Comment:
I tried to motify this test with `User-Agent: bad\\"value`, as if an application had already escaped the dquote.

The result is: `"User-Agent: \"bad\\\\\\\"value\", \"bad\\\\\\\"value\""`.

Is this expected? I would have expected the original value (perhaps quoted), which was already properly escaped.